src/std/text.ab

```diff
@@ -142,8 +142,26 @@ pub fun char_at(text: Text, index: Num): Text {
 }
 
 /// Capitalize the first letter of the given `text`.
+#[allow_absurd_cast]
 pub fun capitalize(text: Text): Text {
-    return trust $ echo "{text}" | sed "s/^\(.\)/\U\1/" $
+    trust {
+        if len(text) == 0: return text
+        const bash_version = $ echo \"\$\{BASH_VERSINFO[0]}\" $ as Num
```

Comment:
It's out of scope for this PR, but we should probably write a builtin returning the Bash version as a `Num`.